lnrpc/.clang-format

```diff
@@ -0,0 +1,7 @@
+---
+Language: Proto
+BasedOnStyle: Google
+IndentWidth: 4
+AllowShortFunctionsOnASingleLine: None
```

Comment:
personal opinion: I think single line short functions are nice and readable. I'm fine with your choice though.

Reply:
This mainly affects how empty messages are displayed (closing bracket on new line or same line). I wanted to imitate my IDE's formatting as close as possible to not have a huge diff to #3996. But I think it's indeed nicer to have it on one line.